doc/source/overview/high_level.md

```diff
@@ -0,0 +1,508 @@
+3. Dummy/Sensitivity: If a variable does not change the output of the model, then it should have attribution 0. If this were not the case, we'd be assigning value to a feature that provides no information.
+4. Additivity/Linearity: The attribution for a feature $x_i$ of a linear composition of two models $f_1$ and $f_2$ given by $c_1 f_1 + c_2 f_2$ is $c_1 a_{1, i} + c_2 a_{2, i}$ where $a_{1, i}$ and $a_{2, i}$ is the attribution for $x_1$ and $f_1$ and $f_2$ respectively.
+
+#### Integrated Gradients
+
+| Model-types | Task-types     | Data-types  |
+| ----------- | -------------- | ----------- |
+| TF/Kera     | Classification | Tabular     |
+|             | Regression     | Image       |
+|             |                | Text        |
+|             |                | Categorical |
+
+
+This method computes the attribution of each feature by integrating the model partial derivatives along a path from a baseline point to the instance. Let $f$ be the model and $x$ the instance of interest. If $f:\mathbb{R}^{n} \rightarrow \mathbb{R}^{m}$ where $m$ is the number of classes the model predicts then let $F=f_k$ where $k \in \{1,..., m\}$. If $f$ is single-valued then $F=f$. We also need to choose a baseline value, $x'$.
+
+$$
+IG_i(x) = (x_i - x_i')\int_{\alpha}^{1}\frac{\partial F (x' + \alpha (x - x'))}{ \partial x_i } d \alpha
+$$
+
+The above sums partial derivatives for each feature over the path between the baseline and instance of interest. In doing so, you accumulate the changes in the prediction that occur due to the changing feature value from the baseline to the instance.
+
+:::{admonition} **Note 5: Choice of Baseline**
+The main difficulty with this method is that as IG is very dependent on the baseline, it's essential to make sure you choose it well. The choice of baseline should capture a blank state in which the model makes essentially no prediction or assigns the probability of each class equally. A common choice for image classification is an image set to black, which works well in many cases but sometimes fails to be a good choice. For instance, a model that classifies images taken at night using an image with every pixel set to black means the attribution method will undervalue the use of dark pixels in attributing the contribution of each feature to the classification. This is due to the contribution being calculated relative to the baseline, which is already dark.
+:::
+
+**Pros**
+- Simple to understand and visualize, especially with image data
+- Doesn't require access to the training data
+
+**Cons**
+- White box method. Requires the partial derivatives of the model outputs with respect to inputs
+- Requires choosing the baseline which can have a significant effect on the outcome (See Note 5)
+
+
+#### KernelSHAP
```

Comment:
Consistency of naming. Let's use `Kernel SHAP` in text, unless referring to the implementation in which case it's `KernelShap`.